src/main/java/ita/softserve/course_evaluation_analytics/service/impl/ChatsMessagesServiceImpl.java

```diff
@@ -0,0 +1,19 @@
+package ita.softserve.course_evaluation_analytics.service.impl;
+
+import ita.softserve.course_evaluation_analytics.entity.ChatsMessages;
+import ita.softserve.course_evaluation_analytics.repository.ChatsMessagesRepository;
+import ita.softserve.course_evaluation_analytics.service.ChatsMessagesService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+@RequiredArgsConstructor
+public class ChatsMessagesServiceImpl implements ChatsMessagesService {
+    private final ChatsMessagesRepository chatsMessagesRepository;
+    @Override
+    public List<ChatsMessages> countMessagesByDays() {
```

Comment:
method name is not valid here, you return a list of smth not the value of objects in the db. please, rename